foreign_cc/built_tools/make_build.bzl

```diff
@@ -29,8 +36,69 @@ def _make_tool_impl(ctx):
+            ctx = ctx,
+            cc_toolchain = cc_toolchain,
+            requested_features = ctx.features,
+            unsupported_features = ctx.disabled_features,
+        )
+        ar_path = cc_common.get_tool_for_action(
+            feature_configuration = feature_configuration,
+            action_name = CPP_LINK_STATIC_LIBRARY_ACTION_NAME,
+        )
+        cc_path = cc_common.get_tool_for_action(
+            feature_configuration = feature_configuration,
+            action_name = C_COMPILE_ACTION_NAME,
+        )
+        ld_path = cc_common.get_tool_for_action(
+            feature_configuration = feature_configuration,
+            action_name = CPP_LINK_EXECUTABLE_ACTION_NAME,
+        )
+        c_compile_variables = cc_common.create_compile_variables(
+            feature_configuration = feature_configuration,
+            cc_toolchain = cc_toolchain,
+        )
+        cflags = cc_common.get_memory_inefficient_command_line(
+            feature_configuration = feature_configuration,
+            action_name = C_COMPILE_ACTION_NAME,
+            variables = c_compile_variables,
+        )
+        c_link_variables = cc_common.create_link_variables(
+            feature_configuration = feature_configuration,
+            cc_toolchain = cc_toolchain,
+        )
+        ldflags = cc_common.get_memory_inefficient_command_line(
+            feature_configuration = feature_configuration,
+            action_name = CPP_LINK_EXECUTABLE_ACTION_NAME,
+            variables = c_link_variables,
+        )
+
+        sysroot_cflags = [flag for flag in cflags if flag.startswith("--sysroot=")]
+        non_sysroot_cflags = [flag for flag in cflags if not flag.startswith("--sysroot=")]
+        sysroot_ldflags = [flag for flag in ldflags if flag.startswith("--sysroot=")]
+        non_sysroot_ldflags = [flag for flag in ldflags if not flag.startswith("--sysroot=")]
+
+        # Make's build script does not forward CFLAGS to all compiler and linker
+        # invocations, so we append --sysroot flags directly to CC.
+        absolute_cc = _absolutize(ctx.workspace_name, cc_path, True)
+        if sysroot_cflags:
+            absolute_cc += " " + _join_flags_list(ctx.workspace_name, sysroot_cflags)
+        absolute_ld = _absolutize(ctx.workspace_name, cc_path, True)
+        if sysroot_ldflags:
+            absolute_ld += " " + _join_flags_list(ctx.workspace_name, sysroot_ldflags)
+
+        env = {
+            "AR": _absolutize(ctx.workspace_name, ar_path, True),
+            "CC": absolute_cc,
+            "CFLAGS": _join_flags_list(ctx.workspace_name, non_sysroot_cflags),
+            "LD": absolute_ld,
+            "LDFLAGS": _join_flags_list(ctx.workspace_name, non_sysroot_ldflags),
+        }
+
+        configure_env = " ".join(["%s=\"%s\"" % (key, value) for key, value in env.items()])
         script = [
-            "./configure --without-guile --with-guile=no --disable-dependency-tracking --prefix=$$INSTALLDIR$$",
```

Comment:
Did you intentionally remove the `—with-guile=no` argument?

Reply:
I didn't, thanks for catching it.